egs/uw3/v1/local/score.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
```

Comment:
Please change this file so that it only calls `steps/scoring/score_kaldi_wer.sh` and `steps/scoring/score_kaldi_cer.sh` with the supplied arguments. You can get help from `hkust/s5/local/score.sh`